lib/bundler/cli/install.rb

```
@@ -185,5 +181,13 @@ def print_post_install_message(name, msg)
       Bundler.ui.confirm "Post-install message from #{name}:"
       Bundler.ui.info msg
     end
+
+    def check_for_options_conflicts
```

this should raise an exception